### PR TITLE
Feat/restore old gifs unfurling

### PR DESCRIPTION
**Pull request by igor-sirotin**
Requires:
- https://github.com/status-im/status-desktop/pull/11940
- https://github.com/status-im/status-desktop/pull/11976

### What does the PR do

Because the new unfurling doesn't yet support GIFs, I've reincarnated old local unfurling. For GIFs only.

### Screenshot of functionality (including design for comparison)

https://github.com/status-im/status-desktop/assets/25482501/e1af0221-4432-42da-b6ba-fba3183f6bfe

### CODE REVIEW

File: ui/imports/shared/views/chat/LinksMessageView.qml [hunk 1]
```diff
@@ -227,9 +229,231 @@ ColumnLayout {
...
+            catch(e) {
+                console.warn("error parsing link preview data", previewData)
+            }
+        }
+    }
+
+    ListModel {
+         id: linksModel
+
+         property var rawData
```
Comment (MishkaRogachev) [inline ui/imports/shared/views/chat/LinksMessageView.qml:285]: i,o putting a raw model in the d would be more convenient

File: ui/imports/shared/views/chat/LinksMessageView.qml [hunk 2]
```diff
@@ -227,9 +229,231 @@ ColumnLayout {
...
+                    }
+                    onClicked: RootStore.setNeverAskAboutUnfurlingAgain(true)
+                    Component.onCompleted: {
+                        background.radius = Style.current.padding;
+                    }
                 }
             }
         }
     }
+
```
Comment (MishkaRogachev) [inline ui/imports/shared/views/chat/LinksMessageView.qml:456]: extra lines

File: ui/imports/shared/views/chat/LinksMessageView.qml [hunk 3]
```diff
@@ -227,9 +229,231 @@ ColumnLayout {
...
+//                    }
+                ]
+            }
+        }
+    }
+
+    Component {
+        id: enableLinkComponent
+
+        Rectangle {
```
Comment (MishkaRogachev) [inline ui/imports/shared/views/chat/LinksMessageView.qml:348]: it would be better to move it to a separate file

**Review by MishkaRogachev — APPROVED** [review]
Great job! Just a few minor questions

File: ui/imports/shared/views/chat/LinksMessageView.qml (hunk 3 above)
Comment (MishkaRogachev) [reply ui/imports/shared/views/chat/LinksMessageView.qml:348]: All right, you can ignore those questions :)

File: ui/imports/shared/views/chat/LinksMessageView.qml [hunk 4]
```diff
@@ -231,4 +234,226 @@ ColumnLayout {
...
+//                    State {
+//                        name: "loadLinkPreview"
+//                        when: unfurl && !isImage && !isStatusDeepLink
+//                        PropertyChanges { target: tempLoader; sourceComponent: unfurledLinkComponent }
+//                    },
+//                    State {
+//                        name: "statusInvitation"
+//                        when: unfurl && isStatusDeepLink
+//                        PropertyChanges { target: tempLoader; sourceComponent: invitationBubble }
+//                    }
```
Comment (jrainville) [inline ui/imports/shared/views/chat/LinksMessageView.qml:339]: Can we get rid of this? Maybe we could keep the `statusInvitation`? Would that make the old component for Status links work?

**Review by jrainville — APPROVED** [review]

File: ui/imports/shared/views/chat/LinksMessageView.qml (hunk 4 above)
Comment (igor-sirotin) [reply ui/imports/shared/views/chat/LinksMessageView.qml:339]: Let's merge this, I'm bringing the fix with another PR

**Pull request merged**